middleware/primary_resource_logic/data_requests.py

```diff
@@ -287,20 +290,34 @@ def get_data_request_related_sources(db_client: DatabaseClient, dto: GetByIDBase
         role=RelationRoleEnum.STANDARD,
         column_permission=ColumnPermissionEnum.READ,
     )
-    results = db_client.get_linked_rows(
-        link_table=Relations.LINK_DATA_SOURCES_DATA_REQUESTS,
-        left_id=int(dto.resource_id),
-        left_link_column="request_id",
-        right_link_column="source_id",
-        linked_relation=Relations.DATA_SOURCES_EXPANDED,
-        linked_relation_linking_column="airtable_uid",
-        columns_to_retrieve=permitted_columns,
+    results = db_client.get_data_requests(
```

Comment:
Excellent job with replacing the `linked_rows` logic with this. There will likely be more cases in the future where linked rows will be used, so we'll see if this logic continues to work there as well.